projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -90,8 +92,10 @@ module.exports = async function (...args) {
 	const runLegacyBuild = !federation || federation.mode !== 'default';
 
 	if (inputPathExists && runLegacyBuild) {
-		if (fs.existsSync('tsconfig.json')) {
-			spawnSync('tsc', ['--noEmit']);
+		const isTypeScript = fs.existsSync('tsconfig.json');
```

Comment:
Last question... I think we mentioned `tsconfig.json` might be auto-generated from within the tool but I'm not seeing that in here... is that still part of the plan for the initial roll-out or are we planning to do manual `tsconfig.json` for now and re-evaluate?

Reply:
![Screen Shot 2021-03-31 at 12 54 00](https://user-images.githubusercontent.com/905006/113140333-32927580-9220-11eb-8ffc-cf0f6cd3b618.png)

![facepalm](https://user-images.githubusercontent.com/905006/113140399-48a03600-9220-11eb-8dd1-e47268f258a1.gif)